config/global-constants/src/lib.rs

```diff
@@ -22,5 +22,5 @@ pub const VALIDATOR_NETWORK_KEY: &str = "validator_network";
 pub const SAFETY_DATA: &str = "safety_data";
 pub const WAYPOINT: &str = "waypoint";
 pub const GENESIS_WAYPOINT: &str = "genesis-waypoint";
-pub const MOVE_MODULES: &str = "move_modules";
+pub const MOVE_MODULES: &str = "module_publish_modules";
```

Comment:
Is this an intended change? Seems out of scope of the smoke-test

Reply:
Oh! That was jetbrains refactor renaming...